packages/amplication-data-service-generator/src/server/resource/service/to-many.template.ts

```diff
@@ -16,6 +16,6 @@ export class Mixin {
   ): Promise<RELATED_ENTITY[]> {
     return this.prisma.DELEGATE.findUnique({
       where: { id: parentId },
-    }).PROPERTY(args);
+    }).PROPERTY(args) as Promise<RELATED_ENTITY[]>;
```

Comment:
I would always recommend avoiding casting wherever possible. 
How about returning empty array `[]` when prisma returns `null`?! 

```suggestion
return await this.prisma.DELEGATE.findUnique({
      where: { id: parentId },
    }).PROPERTY(args) ?? [];
```

Reply:
I really like this idea